src/theory/datatypes/theory_datatypes_type_rules.h

```diff
@@ -104,22 +105,6 @@ struct DatatypeConstructorTypeRule {
         return false;
       }
     }
-    //if we support subtyping for tuples, enable this
```

Comment:
Say in the PR description why you can delete this code?

Reply:
Done